docs/topics/forms/index.txt

```diff
@@ -686,6 +686,32 @@ Useful attributes on ``{{ field }}`` include:
 
         <label for="id_email">Email address:</label>
 
+``{{ field.legend_tag }}``
+
+    .. versionadded:: 4.1
+
+    Similar to ``field.label_tag`` but uses a ``<legend>`` tag in place of
+    ``<label>``.
+
+``{{ field.use_fieldset }}``
+
+    .. versionadded:: 4.1
+
+    This attribute is ``True`` if the form field's widget contains
+    multiple inputs and should be wrapped in a ``<fieldset>`` to improve
+    accessibility. An example use in a template:
```

Comment:
```suggestion
    multiple inputs and should be wrapped in a ``<fieldset>`` with a ``<legend>``
    to improve accessibility. An example use in a template:
```

Do we prefer the generic "improve accessibility" or explain further what the fieldset achieves compared to other groupings? Not a fan of this wording but here’s an attempt:

```suggestion
    multiple inputs that should be semantically grouped in
    a ``<fieldset>`` with a ``<legend>`` to improve accessibility.
    An example use in a template:
```